.editorconfig

```diff
@@ -8,5 +8,5 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 charset = utf-8
-indent_style = space
+indent_style = tab
```

Comment:
please don't change these

Reply:
I updated it because all files seems to be indented with tabulation and the editorconfig weren’t reflecting that.
I’ll revert it then.